.github/workflows/qaqc.yaml

```diff
@@ -0,0 +1,57 @@
+name: QualityContol
+
+on:
+  pull_request:
+    paths:
+      - 'book/**'
+      - '.github/workflows/qaqc.yaml'
+    branches:
+      - main
+
+env:
+  DOCKER_IMAGE: uwhackweek/snowex:latest
+
+jobs:
+  quality-control:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Ensure Clean Notebooks
+      continue-on-error: true
+      run: |
+        pip install nb-clean==2.0.2
+        find book -name "*ipynb" | xargs nb-clean check --preserve-cell-metadata
+
+    # NOTE: this isn't a comprehensive spellcheck, just common typos
+    - name: Spellcheck
+      continue-on-error: true
+      uses: codespell-project/actions-codespell@master
+      with:
+        skip: .github/workflows/qaqc.yaml,./book/tutorials/geospatial/geospatial_cookbook_rendered.ipynb
+        ignore_words_list: hist,MAPE,te
```

Comment:
can remove these lines. i dont think the ignore_words_list is working for some reason...